sample-app/src/hooks/useComponentMountStatus.tsx

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useRef } from "react";
+
+export default function useComponentMountStatus() {
```

Comment:
Can we add a comment saying that this is used to suppress react errors that do not need to be suppressed in the next version of react, and include a link to the react pr?